archivers/bzip3/Portfile

```
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        kspalaiologos bzip3 1.2.1
+github.tarball_from releases
+categories          archivers
+license             LGPL-3
+
+maintainers         {@sech1p gmail.com:sech1piam} openmainter
```

You misspelled "openmaintainer".
```suggestion
maintainers         {@sech1p gmail.com:sech1piam} openmaintainer
```